### PR TITLE
Change instances of machineCIDR to machineNetwork.cidr

### PR DESCRIPTION
**Pull request by jboxman**
Now that it's possible to do this: https://issues.redhat.com/browse/OSDOCS-877.

### CODE REVIEW

**Comment by jboxman** [comment]
@kalexand-rh - this change touches a bunch of install modules.
@maxwelldb - this change touches some OSP modules.

I'm not sure the best candidates from QE for validation.

**Comment by kalexand-rh** [comment]
@jianlinliu, since this change touches so many areas of installation, will you please let us know what QE reviews you think are most appropriate?

File: machine_management/manually-scaling-machineset.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ see xref:../machine_management/modifying-machineset.adoc#modifying-machineset[Mo
 
 .Prerequisites
 
-* If you enabled the cluster-wide proxy and scale up workers not included in `networking.machineCIDR` from the installation configuration, you must xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[add the workers to the Proxy object's `noProxy` field] to prevent connection issues.
+* If you enabled the cluster-wide proxy and scale up workers not included in `networking.machineNetwork.cidr` from the installation configuration, you must xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[add the workers to the Proxy object's `noProxy` field] to prevent connection issues.
```
Comment (jianlinliu) [inline machine_management/manually-scaling-machineset.adoc]: Strictly to speak, I do not think `networking.machineCIDR` is replaced by `networking.machineNetwork.cidr`, but `networking.machineNetwork`, and the key accepts an array, items in this array is key=value pair.

Comment (jianlinliu) [reply machine_management/manually-scaling-machineset.adoc]: Just like the following example shows, `networking.machineNetwork[0].cidr` should be more accurate.

Comment (jianlinliu) [reply machine_management/manually-scaling-machineset.adoc]: proxy? I think it care about the whole array list from networking.machineNetwork's value

Comment (gpei) [reply machine_management/manually-scaling-machineset.adoc]: yes, all the values of cidr in networking.machineNetwork would be added into `noProxy`

Comment (jboxman) [reply machine_management/manually-scaling-machineset.adoc]: Thanks!

File: machine_management/manually-scaling-machineset.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ see xref:../machine_management/modifying-machineset.adoc#modifying-machineset[Mo
 
 .Prerequisites
 
-* If you enabled the cluster-wide proxy and scale up workers not included in `networking.machineCIDR` from the installation configuration, you must xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[add the workers to the Proxy object's `noProxy` field] to prevent connection issues.
+* If you enabled the cluster-wide proxy and scale up workers not included in `networking.machineNetwork[0].cidr` from the installation configuration, you must xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[add the workers to the Proxy object's `noProxy` field] to prevent connection issues.
```
Comment (jianlinliu) [inline machine_management/manually-scaling-machineset.adoc]: Should be `networking.machineNetwork[].cidr`, right?

File: modules/installation-configure-proxy.adoc [hunk 3]
```diff
@@ -26,7 +26,7 @@ ifdef::bare-metal[]
 [NOTE]
 ====
 For bare metal installations, if you do not assign node IP addresses from the
-range that is specified in the `networking.machineCIDR` field in the
+range that is specified in the `networking.machineNetwork[0].cidr` field in the
```
Comment (jianlinliu) [inline modules/installation-configure-proxy.adoc]: Should be `networking.machineNetwork[].cidr`, right?

File: modules/installation-configure-proxy.adoc [hunk 4]
```diff
@@ -39,7 +39,7 @@ endif::bare-metal[]
 +
 [NOTE]
 ====
-The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineCIDR`, `networking.clusterNetwork.cidr`, and `networking.serviceNetwork` fields from your installation configuration.
+The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineNetwork[0].cidr`, `networking.clusterNetwork[0].cidr`, and `networking.serviceNetwork` fields from your installation configuration.
```
Comment (jianlinliu) [inline modules/installation-configure-proxy.adoc]: Should be `networking.machineNetwork[].cidr` and `networking.clusterNetwork[].cidr`, right?

File: networking/enable-cluster-wide-proxy.adoc [hunk 5]
```diff
@@ -18,7 +18,7 @@ The cluster-wide proxy is only supported if you used a user-provisioned infrastr
 +
 [NOTE]
 ====
-The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineCIDR`, `networking.clusterNetwork.cidr`, and `networking.serviceNetwork` fields from your installation configuration.
+The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineNetwork[0].cidr`, `networking.clusterNetwork[0].cidr`, and `networking.serviceNetwork` fields from your installation configuration.
```
Comment (jianlinliu) [inline networking/enable-cluster-wide-proxy.adoc]: Should be `networking.machineNetwork[].cidr` and `networking.clusterNetwork[].cidr`, right?

**Comment by jboxman** [comment]
@jianlinliu, updated.

File: modules/installation-configure-proxy.adoc [hunk 6]
```diff
@@ -39,7 +39,7 @@ endif::bare-metal[]
 +
 [NOTE]
 ====
-The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineCIDR`, `networking.clusterNetwork.cidr`, and `networking.serviceNetwork` fields from your installation configuration.
+The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineNetwork[].cidr`, `networking.machineNetwork[].cidr`, and `networking.serviceNetwork` fields from your installation configuration.
```
Comment (jianlinliu) [inline modules/installation-configure-proxy.adoc]: Two `networking.machineNetwork[].cidr` here? duplicated? one of them should be `networking.clusterNetwork[].cidr`, right?

Comment (jboxman) [reply modules/installation-configure-proxy.adoc]: @jianlinliu, oops, I'll fix that!

**Comment by jianlinliu** [comment]
LGTM.

**Pull request merged**